### PR TITLE
Incorrect SITE_URL value is used when the website is built (mozilla#907)

### PR DESCRIPTION
**Pull request by anuragakella**
@Rob--W I'd like to work on this.
Added a SITE_URL to `.circleci/config.yml`
I'm new here, so can someone please let me know if my commit was correct?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ jobs:
       - image: circleci/node:10.18.1@sha256:8230f70c03b6ecf521a34125fb090ec4b47068205454e5820859c00260c9bd23
     environment:
       ADDON_URL: https://addons.mozilla.org/firefox/addon/firefox-color/
+      SITE_URL: https://color.firefox.com/
```
Comment (Rob--W) [inline .circleci/config.yml]: This is not the correct location.

The file defines different behaviors for different branches (#907 has documented the URLs, those are also mentioned in https://github.com/mozilla/FirefoxColor#environment-list). The section of the `config.yml` file that you're editing applies to all jobs.

You should figure out how to define environment variables for the branch-specific jobs in the config file. Try to look for examples in other repositories, and/or the documentation at https://circleci.com/docs/2.0/env-vars/ to learn how to do so.

Comment (anuragakella) [reply .circleci/config.yml]:
```
jobs:

  build_and_test:
    branch: development
      environment:
          SITE_URL: https://mozilla.github.io/FirefoxColor/
    branch: stage
      environment:
          SITE_URL: https://color.stage.mozaws.net/
    branch: production
      environment:
          SITE_URL: https://color.firefox.com/
    docker:
      - image: circleci/node:10.18.1@sha256:8230f70c03b6ecf521a34125fb090ec4b47068205454e5820859c00260c9bd23
    environment:
      ADDON_URL: https://addons.mozilla.org/firefox/addon/firefox-color/
    working_directory: ~/FirefoxColor
    steps:
``` 
...
Would this work?

Comment (Rob--W) [reply .circleci/config.yml:10]: Probably not. I think that fixing this bug is not a good first bug. So unless you are already familiar with CircleCI configuration, I suggest to not work on this bug.

Comment (anuragakella) [reply .circleci/config.yml:10]: Alright, understood. 
Can you suggest any bugs that I could fix? 
Thank you.

**Comment by Rob--W** [comment]
The bug has been resolved, so I'm closing this issue.

I saw your email about 866; I'll get to that this week.

**Pull request closed without merging**